charmcraft/commands/utils.py

```diff
@@ -29,3 +30,16 @@ def make_executable(fh):
     mode_x = mode_r >> 2
     mode = mode | mode_x
     os.fchmod(fileno, mode)
+
+
+def parse_os_release(filename="/etc/os-release"):
```

Comment:
Docstring and meaningful variable names, please.

Reply:
ack (dropping it entirely instead)